sources/packages/web/src/assets/css/vuetify.scss

```diff
@@ -149,3 +149,7 @@
 .v-field {
   overflow: inherit !important;
 }
+
+.v-table {
```

Comment:
IMO if it is only for a one-page v-table. better to custom class specific to the page. if it's the same across the project. then it's good. but I think v-table can be used for different purposes,

Reply:
It is same across the object. Otherwise it is setting font color to 00000 (black)